js/app/modules/selection/relatedArticles.js

```diff
@@ -0,0 +1,47 @@
+/* exported RelatedArticles */
+
+// Copyright 2017 Endless Mobile, Inc.
+
+const Eknc = imports.gi.EosKnowledgeContent;
+
+const Module = imports.app.interfaces.module;
+const HistoryStore = imports.app.historyStore;
+const ReadingHistoryModel = imports.app.readingHistoryModel;
+const Xapian = imports.app.modules.selection.xapian;
+const Pages = imports.app.pages;
+
+/**
+ * Class: RelatedArticles
+ *
+ * Retrieves articles that are tagged with at least one of the tags from
```

Comment:
This is not going to be the actual functionality of this module after we add related articles to the data model, right?